resources/js/Pages/Lessons/Show.vue

```diff
@@ -0,0 +1,98 @@
+<script setup>
+import {onMounted, ref} from "vue";
+import unityGameSetUp from '../../GameViewJava';
+import {Head, Link} from '@inertiajs/inertia-vue3';
+
+onMounted(() => {
+    unityGameSetUp();
+    window.addEventListener("stateFromUnity",(e) => receivedJsonSaveFromUnity(e));
+    window.addEventListener("requestData", (e) => requestLoadGameData(e));
+    window.addEventListener("StartChallenge", (e) => receivedChallengeFromUnity(e));
+})
+
+const props = defineProps({
+    lesson: Object ,
+    course: Object,
+})
+
+let showChallengeWindow = ref(false);
+let resultWarning = ref(false);
+let challengeUserAnswer = ref("");
+let challengeAnswer = ref("");
+
+const receivedChallengeFromUnity = async (e) => {
+    challengeAnswer.value = e.detail.challenge;
+    console.log(challengeAnswer.value);
+    showChallengeWindow.value = true;
+}
+
+const checkAnswer = (e) => {
+    if(challengeUserAnswer.value.trim() === challengeAnswer.value){
+        showChallengeWindow.value = false;
+        gameInstance.SendMessage('GameController','CorrectAnswer');
+    }
+    else {
+        resultWarning.value = true;
+    }
+}
+
+const receivedJsonSaveFromUnity = async (e) => {
+
+    try {
+        await axios.post(route('courses.lessons.save',
+                {course: props.course, lesson:props.lesson} ),
+            {game_state: e.detail.gamestate });
+    } catch (e) {
+        console.log("failed to add to databse, error: ", +e);
+    }
+}
+
+const requestLoadGameData = async (e) => {
+    try {
+        let response = await axios.get(route('courses.lessons.load', {course:props.course, lesson: props.lesson}))
+        let gameMessage = response.data['gameLoaded'];
+        if (response.data['gameLoaded'] == null) {
```

Comment:
Here you can add a blank line above the if statement
```suggestion
        let response = await axios.get(route('courses.lessons.load', {course:props.course, lesson: props.lesson}))
        let gameMessage = response.data['gameLoaded'];
        
        if (response.data['gameLoaded'] == null) {
```